packages/codd/scripts/codd.py

```
@@ -343,6 +343,182 @@ def cmd_validate(root: Path, config: cc.CoddConfig) -> int:
     return 1 if errors else 0
 
 
+# ---------------------------------------------------------------------------
+# impact（Issue #94 / 信頼度3帯域）
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)